junebug/tests/test_api.py

```diff
@@ -57,14 +57,26 @@ def test_http_error(self):
 
     @inlineCallbacks
     def test_get_channel_list(self):
+        redis = yield self.get_redis()
+        config = self.default_channel_config
+
         resp = yield self.get('/channels')
-        yield self.assert_response(
-            resp, http.INTERNAL_SERVER_ERROR, 'generic error', {
-                'errors': [{
-                    'message': '',
-                    'type': 'NotImplementedError',
-                    }]
-                })
+        yield self.assert_response(resp, http.OK, 'channels listed', [])
+
+        yield Channel(redis, {}, config, u'test-channel-1').save()
+
+        resp = yield self.get('/channels')
+        yield self.assert_response(resp, http.OK, 'channels listed', [
+            u'test-channel-1',
```

Comment:
Mostly my thought was that a list of UUIDs was not that useful, and that it would be nice to add the names at least. If we can add more data easily once we have the names, then I'm +1 on adding that (or even just including the complete data if we have it).